jobs/service-fabrik-broker/templates/bin/service-fabrik-broker_ctl.erb

```diff
@@ -27,18 +27,33 @@ case $1 in
         if ! grep -Fxq "\$ModLoad imtcp" /etc/rsyslog.d/10-fabrik_modules.conf ; then echo "\$ModLoad imtcp" >> /etc/rsyslog.d/10-fabrik_modules.conf ; fi
```

Comment:
dont we want to do this for all the jobs that run on nodejs ?

Reply:
We can do that. I did it only on broker because that was going high on CPU. If we add everywhere then there is no selective toggle.

Reply:
Ok , we can go this way for now , then we can consider for other jobs later..